README.md

```diff
@@ -209,6 +209,24 @@ select * from {{ ref('users') }}
 {% endif %}
 ```
 
+##### `merge`
+
+Through the `merge` incremental strategy, dbt-trino constructs `MERGE` statement which `INSERT` new records and `UPDATE` existing ones based on unique key specified by `unique_key`. 
+
+```jinja2
+{{
+    config(
+      materialized = 'incremental',
+      unique_key='user_id',
+      incremental_strategy='merge',
+      )
+}}
+select * from {{ ref('users') }}
```

Comment:
Please do add also an example with `incremental_predicates`

Reply:
I disabled it from macro since it's a part of the dbt refactor which is not yet out.